torch/csrc/jit/codegen/cuda/scheduler/compile_time_info.h

```diff
@@ -24,6 +25,8 @@ namespace HeuristicCompileTime {
 
 //! Enum for all possible types of cached entries of compile-time info.
 enum class CompileTimeEntryType {
+  DOMAIN_MAP,
+  REFERENCE_TENSORS,
```

Comment:
Would appreciate more specific namings for the new entries. 

`DOMAIN_MAP` probably ok if it can expose both `ca_map` and `root_map` to all schedulers.

`REFERENCE_TENSORS` sounds quite likely to have naming collision with other schedulers.

Reply:
`REFERENCE_TENSORS` is a vector of tensors, and it is intended to be shared by many schedulers. For example, the pointwise scheduler can cache `{reference_tv}` and the transpose scheduler can cache `{reference1, reference2}`